pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2546,6 +2546,45 @@ protected void internalSetMaxTopicsPerNamespace(Integer maxTopicsPerNamespace) {
         internalSetPolicies("max_topics_per_namespace", maxTopicsPerNamespace);
    }
 
+   protected void internalSetProperty(String key, String value) {
+       validatePoliciesReadOnlyAccess();
+       try {
+           final String path = path(POLICIES, namespaceName.toString());
+           updatePolicies(path, (policies) -> {
+               policies.properties.put(key, value);
```

Comment:
What if I want to remove this key?

Reply:
Do you mean here https://github.com/apache/pulsar/pull/10015/files#diff-8696b1e2ba4ee355256e31b33ccc4f1cab77f1c6c79f7b4ab508b73d43d8ad55R2576 ?

Reply:
Oh i see, thanks for the explanation